app/js/instances/instanceDetailController.js

```diff
@@ -83,6 +81,14 @@ function instanceDetailController($scope, $routeParams, $timeout, flux, dockerSe
             }
         });
     };
+
+    $scope.removeImage = function(image) {
+        image.loading = true;
+        dockerService.d('images.remove', {
+            host: $routeParams.host,
```

Comment:
Hmmm that's odd. I was able to get valid output with this. Double check the spelling :)

```
$scope.removeImage = function(image) {
        console.log($scope.instance);
        console.log($scope.instance.alias);

        image.loading = true;
        ....
```